packages/dds/test-dds-utils/src/gcTestRunner.ts

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { ISharedObject } from "@fluidframework/shared-object-base";
+
+/**
+ * Defines a set of functions to be passed to the GC test runner.
+ */
+export interface IGCTestProvider {
+    /** Returns the DDS whose GC data is to be verified */
+    getSharedObject(): ISharedObject;
```

Comment:
we usually implement it as readonly property - useful to have it that way for consistency with rest of code